packages/vscode-extension/PRERELEASE.md

```diff
@@ -1,7 +1,38 @@
-# Teams Toolkit v5.0 Pre-release
+# Teams Toolkit Pre-release
 
 ## Changelog
 
+### July 18, 2023
+
+New features:
+
+- Import an existing SharePoint Framework solution and continue development with Teams Toolkit
+    ![SPFx Existing App](https://github.com/OfficeDev/TeamsFx/assets/11220663/3944f5c8-6c8c-4b4d-8df8-dc4f45b5967f)
+- A new link unfurling app template to help you get started with displaying rich content from links in Teams messages and Outlook emails.
+    ![Link Unfurling](https://github.com/OfficeDev/TeamsFx/assets/11220663/6e8b982a-0531-4ec1-8420-f6f17955ff40)
+
+Enhancement:
+
+- Updated `React with Fluent UI` app template to use OBO flow for SSO implenmentation to improve SSO experience in mobile platforms.
+- Updated project scaffold success notification for Outlook Add-in.
+
+TeamsFx SDK ([`@microsoft/teamsfx`](https://www.npmjs.com/package/@microsoft/teamsfx)) `v2.2.2`:
+
+- Rename existing internal class `ConversationReferenceStore` to `DefaultConversationReferenceStore`.
+- Deprecate existing interface `NotificationStorage`.
+- Add new interface `ConversationReferenceStore` with pagination API support.
+- Add new options for users to set customized conversation reference store, and deprecate existing option to set customize notification target storage.
```

Comment:
Added a new option to set a custom `ConversationReferenceStore` and deprecated the existing option for `NotificationStorage`.

Where do you set the custom `ConversationReferenceStore`?